### DISCUSSION TITLE
快捷键方面建议

### DISCUSSION THREAD

**Discussion by Akihie** [jaywcjlove/reference#834] [Ideas]
我感觉如果左解释，右边快捷键可能会更好一点，毕竟阅读习惯是从左到右。而且是从文字找快捷键。这只是我这方面的感觉，大佬可以多问几个人。

**Comment by jaywcjlove** [comment]
@Akihie 指的是搜索的地方吗？

**Reply by Akihie** [reply]
展示方面
<img width="922" alt="image" src="https://github.com/user-attachments/assets/f69c6db7-677d-40d6-a502-7e88e5f9e9c8">
这两个换下位置。这只是我的感觉。可以问下别人，

**Reply by jaywcjlove** [reply]
https://github.com/jaywcjlove/reference/commit/88559b491cff8053188fb5c6e2f6e8ca33ea08b9

我已经实现了后面显示快捷键的方式，但是工作量有点大，欢迎提交 PR
